shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -79,10 +73,9 @@ public static void startInitialization(
    */
   public static void ensureInitializationComplete(
       @NonNull Context applicationContext, @Nullable String[] args) {
-    if (isRunningInRobolectricTest) {
-      return;
-    }
-    FlutterLoader.getInstance().ensureInitializationComplete(applicationContext, args);
+    FlutterInjector.instance()
```

Comment:
This is a breaking change... Apps testing via robolectric will need to do something else to avoid the native library load such as using a mockito spy and overriding loadNativeLibrary.

Reply:
Ya, I thought it might be as well, though I'm not seeing anyone using it cs/isrunninginrobolectrictest. Am I searching it wrong? I'm happy to let https://github.com/flutter/engine/pull/20789/files#diff-8e0c9afcddc7151854497b73e0484fccR92 read out of something in the flutterinjector as well.

Reply:
I am not against the change :) We just need to document it. You are right, I don't think g3 is using this but someone else might.

Reply:
Our new thing is that breaking changes are things that break tests 😐  https://flutter.dev/docs/resources/compatibility

Reply:
🤷 Up to you :)